### DISCUSSION TITLE
Cannot get DACAI screen to update

### DISCUSSION THREAD

**Discussion by KobraNeo** [mriscoc/Ender3V2S1#893] [Q&A]
Ender 3 S1. F4 UBL firmware is installed on the printer, but I cannot get the DACAI screen to update using either package. I'm afraid of damaging my micro sd card reader in the screen from all the ejecting.
 Any advice would be greatly appreciated.

**Comment by mriscoc** [comment]
Check: https://github.com/mriscoc/Ender3V2S1/wiki/How-to-generate-a-gcode-preview#dacai-screen-units

**Reply by KobraNeo** [reply]
You may see my reply on another comment, but it turns out I have the TJC screen so all of this has been for naught. Firmware on the printer works great though, thank you so much for it!
